bin/local-dev/wordpress/Dockerfile

```diff
@@ -4,6 +4,7 @@ FROM wordpress:${WP_VERSION}-php7.4-apache
 
 ENV NODE_VERSION 10.18.1
 ENV YARN_VERSION 1.21.1
+ENV XDEBUG_VERSION 2.9.8
```

Comment:
Pin the Xdebug version as it installs the latest stable v3 by default which isn't compatible with the config we have for this container.